.buildkite/pipelines/performance/nightly.yml

```diff
@@ -19,10 +19,10 @@ steps:
       queue: c2-16
     key: build
 
-  - label: ":muscle: Performance Tests"
-    command: .buildkite/scripts/steps/functional/performance.sh
+  - label: ":muscle: Performance Tests with Playwright config"
+    command: .buildkite/scripts/steps/functional/performance_playwright.sh
```

Comment:
Can we move the steps from this file directly into this file? Or do they need to be separate for some reason?